### ISSUE TITLE
QuickPick does not read out placeholder

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51824]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10

Additional Details:
NVDA 2018.2,JAWS18.0.5038,MAS1.3.1
same issue is observed in JAWS

**Repro Steps:**
1)Launch VSCode.
2)Navigate to Settings icon in Activity Bar->Press Enter->Select "Color Theme" from dropdown.
3)Check with NVDA reading.

**Actual:**
NVDA does not read the placeholder present on the screen.

**Expected:**
NVDA should read the placeholder which is present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

](url)

[**User] Impact:**
User will not able to know that there is placeholder present  on the screen.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}
](url)

Attachment for Reference:
[[Accessibility]A11y_VSCode_ColorTheme_Screenreade.pptx](https://github.com/Microsoft/vscode/files/2100897/Accessibility.A11y_VSCode_ColorTheme_Screenreade.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This works fine on mac with VoiceOver, but not on windows.
Not sure if this is low hanging fruit, leaving to @chrmarti and @bpasero to decide

**Comment by bpasero** [comment]
That sounds like a bug in NVDA to me, but maybe we can tweak the ARIA label for inputs. We could also see if there is a `aria-placeholder` attribute to use but then we should use it everywhere.

**Comment by isidorn** [comment]
@bpasero would you be willing to do some tweaking for this?
We need it for a better accessibility grade. 
Anyways assigning to October so Christof looks once he is back if you do not have time this week

**Comment by bpasero** [comment]
Probably no time.

**Comment by bpasero** [comment]
I looked into it and have not good explanation for the behaviour. This seems to be triggered by the fact that the input box has a `aria-activedescendant` pointing to a tree/list item below. If I remove that, the placeholder is correctly announced. 

Our only hope may be to announce the placeholder using `aria.alert()` which is ugly too. 

We always used the `aria-activedescendant` trick since the beginning and I cannot remember we had this issue before so we could also try to reach out to NVDIA folks (@alexandrudima I think you did that for the editor a while ago, maybe you know the name still)

**Comment by isidorn** [comment]
@bpasero you can always reach out to this repo here https://github.com/nvaccess/nvda
Or just find the top commiter and ping him here :)

**Comment by chrmarti** [comment]
This works for me with QuickOpen, but not with QuickInput. Both use `aria-activedescendant`, I wonder what the difference is.

**Comment by bpasero** [comment]
@chrmarti maybe tree vs list ARIA roles?

**Comment by isidorn** [comment]
Yes might be. @chrmarti not that I changed one of them to use `listitem` which might be a bad change. Now I am checking the behavior on windows actually..

**Comment by chrmarti** [comment]
You're right, with your change reverted it no longer works with QuickOpen.

**Comment by chrmarti** [comment]
Not focusing any element in the list lets it read the placeholder. This was supposed to be fixed with #57501 which checks if it is in screen reader mode before setting the focus, but there is a bug that still makes it focus an element in most cases. Will fix.

**Comment by chrmarti** [comment]
Actually should have worked before when screen reader mode was on. Closing as duplicate.
